plugin/storage/grpc/options.go

```diff
@@ -37,6 +37,7 @@ const (
 // to bind them to command line flags
 type Options struct {
 	Configuration config.Configuration `mapstructure:",squash"`
+	ConfigV2      config.ConfigV2      `mapstructure:",squash"`
```

Comment:
I recommend leaving Options only as a container for Configuration. It's still needed to support v1 CLI parsing, but in the factory when time comes to start creating services we can convert it to ConfigV2 once.